cime_config/config_component.xml

```diff
@@ -307,6 +307,8 @@
     <valid_values></valid_values>
     <default_value> </default_value>
     <values match="last" modifier="additive">
+      <!-- Turn on chemistry if not using physics testbed -->
+      <value compset="^((?!_CAM%PHYSTEST).)*$">-lmusica-fortran -lmusica -lyaml-cpp</value>
```

Comment:
We ended up deciding to not include these flags until there was a version of MUSICA we could link to on Derecho, as it has been causing difficulties with other people's workflows.  For a slightly more detailed discussion see issue #237.  

Of course this was assuming that most current CAM-SIMA MUSICA development would be done with the MUSICA dockerfile (where these flags are being set automatically), but if that is not the case just let me know and we can figure out an alternative solution.  Thanks!

Reply:
removed!